integration-tests/utils/terra-envs.js

```diff
@@ -2,27 +2,24 @@ module.exports = {
   dev: {
     billingProject: 'saturn-integration-test-dev',
     billingProjectAzure: 'mnolting20231030_azBpDev-1',
```

Comment:
What are these column name/id/table name variables that you're deleting?

Reply:
Oh yeah, I meant to add a comment about these...

This file contains information about environment-specific fixtures used by tests. Since the alpha environment no longer exists, we can remove configuration for it.

The `snapshotColumnName`, `snapshotId`, and `snapshotTableName` values are all unused. They were added for a test that no longer exists.